src/com/lftechnology/java/training/sanish/application/view/employee/EmployeeSearchResultPage.java

```diff
@@ -0,0 +1,63 @@
+package com.lftechnology.java.training.sanish.application.view.employee;
+
+import com.lftechnology.java.training.sanish.application.component.Constants;
+import com.lftechnology.java.training.sanish.application.model.domain.UserEmployee;
+import com.lftechnology.java.training.sanish.application.utility.OutputFormatter;
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Render employee search result page
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class EmployeeSearchResultPage {
+    private static final Logger LOGGER = Logger.getLogger(EmployeeSearchResultPage.class.getName());
+
+    private EmployeeSearchResultPage() {
+    }
+
+    /**
+     * Render employee search result page
+     *
+     * @param userEmployeeList {@link List<UserEmployee>}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void renderPage(List<UserEmployee> userEmployeeList) {
+        String pageContent = OutputFormatter.getFormattedPageTitle(Constants.EMPLOYEE_SEARCH_PAGE);
+        pageContent += "\n Search User list \n";
+        if (userEmployeeList.size() > 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1155)